instrumentation/spring/spring-boot-resources/library/src/test/resources/bootstrap-multi.yml

```
@@ -0,0 +1,8 @@
+spring:
+  application:
+    name: cat-store-bootstrap
+
```

You need to use the `---` separator to have separate documents in a single file
```suggestion

---
```

---

Fixed, thank you very much